pkg/apis/steward/v1alpha1/run_types.go

```diff
@@ -72,6 +72,10 @@ type PipelineSpec struct {
 	// Jenkinsfile Runner.
 	// +optional
 	RunDetails *PipelineRunDetails `json:"runDetails,omitempty"`
+
+	// Profiles defines a set of aspects which are restricted to a named profile
+	// for this aspect.`
+	Profiles *Profiles `json:"profiles,omitempty"`
```

Comment:
Don't repeat the type's documentation.
```suggestion
	Profiles *Profiles `json:"profiles,omitempty"`
```